common/wscript

```diff
@@ -47,6 +47,7 @@ def build(bld):
         'JackActivationCount.cpp',
         'JackAPI.cpp',
         'JackClient.cpp',
+		'JackMetadata.cpp',
```

Comment:
wrong whitespace, likely will result in issues due to python not liking to mix those

Reply:
fixed on d5a4ba6